src/pages/basic_data/institution_admin/person_register/PersonTableRow.tsx

```diff
@@ -235,137 +240,141 @@ export const PersonTableRow = ({
+                                  error={touched && !!error}
+                                  helperText={touched && error}
+                                />
+                              )}
+                            </Field>
+                          </Box>
+                          <Box display={{ display: 'flex', gap: '1rem' }}>
+                            <StartDateField
+                              fieldName={`${employmentBaseFieldName}.startDate`}
+                              disabled={isSubmitting || !hasFetchedPositions}
+                              maxDate={
+                                values.employments[employmentIndex].endDate
+                                  ? new Date(values.employments[employmentIndex].endDate)
+                                  : undefined
+                              }
+                            />
+
+                            <Field name={`${employmentBaseFieldName}.endDate`}>
+                              {({ field, meta: { error, touched } }: FieldProps<string>) => (
+                                <DatePicker
+                                  disabled={isSubmitting || !hasFetchedPositions}
+                                  label={t('common.end_date')}
+                                  PopperProps={{
+                                    'aria-label': t('common.end_date'),
+                                  }}
+                                  value={field.value ? field.value : null}
+                                  onChange={(date) => setFieldValue(field.name, date ?? '')}
+                                  inputFormat="dd.MM.yyyy"
+                                  views={['year', 'month', 'day']}
+                                  mask="__.__.____"
+                                  minDate={
+                                    values.employments[employmentIndex].startDate
+                                      ? new Date(values.employments[employmentIndex].startDate)
+                                      : undefined
+                                  }
+                                  renderInput={(params) => (
+                                    <TextField
+                                      {...params}
+                                      variant="filled"
+                                      error={touched && !!error}
+                                      helperText={<ErrorMessage name={field.name} />}
+                                    />
+                                  )}
+                                />
+                              )}
+                            </Field>
                           </Box>
+                          <Button
+                            disabled={isSubmitting || !hasFetchedPositions}
+                            color="error"
+                            variant="outlined"
+                            onClick={() => {
+                              toggleConfirmDeleteDialog();
+                            }}
```

Comment:
kan godt forenkle denne også, i samme slengen
```suggestion
                            onClick={toggleConfirmDeleteDialog()}
```